requirements/libraries.txt

```diff
@@ -5,51 +5,35 @@ dj-database-url==0.4.2
 djangorestframework==3.6.4
 djangorestframework-csv==2.0.0
 django-csp==3.4
-django-braces==1.0.0
-django-extensions==2.1.0
+django-extensions==2.1.3
 django-flags==3.0.2
 django-haystack==2.7.0
-django-localflavor==1.1
+django-localflavor==2.1
 # django-mptt is required by teachers-digital-platform
 django-mptt==0.9.0
 django-overextends==0.4.3
-# django-storages 1.6.6 drops support for Django 1.8.
-# https://github.com/jschneier/django-storages/blob/master/CHANGELOG.rst
-# Versions >1.5.0,<=1.6.5 fail due to this issue:
-# https://github.com/jschneier/django-storages/issues/382
-django-storages==1.5.0
-# django-taggit 0.23.0 requires Django 1.11. Normally django-taggit is pulled
-# in by Wagtail's setup.py, but Wagtail 1.13 specifies >0.20,<1.0, which
-# unfortunately includes this version. This line can be removed once we are
-# on Django 1.11.
-django-taggit==0.22.2
+django-storages==1.7.1
 django-treebeard==3.0
-django-watchman==0.13.1
-edgegrid-python==1.0.10
+django-watchman==0.15.0
+edgegrid-python==1.1.1
 elasticsearch==2.4.1
-geocoder==1.12.0
 govdelivery==1.2
-html5lib==0.9999999
 Jinja2==2.7.3
 jsonfield==2.0.2
-jsonschema==2.0.0
 lxml==4.1.0
 Markdown==2.6.11
 markdown-emdash==0.1.0
 ntplib==0.3.3
 openpyxl==2.5.0
 pyelasticsearch==0.6.1
-python-dateutil==2.1
-PyYAML==3.11
```

Comment:
I agree that this should be removed, as it's not needed anywhere in cfgov-refresh. It is required by college-costs and included [in its setup.py](https://github.com/cfpb/college-costs/blob/master/setup.py#L16) with `>=3.11,<3.14`.

Ideally, we could remove this and college-costs would bring it in as part of its `pip install`. But, like django-mptt, which is also included here because it is required by TDP (#4461), if we don't include PyYAML, then college-costs might not work right when run in Docker.

This is because the docs [recommend](https://github.com/cfpb/cfgov-refresh/blob/master/docs/usage.md#develop-satellite-apps) cloning satellite repos into a local develop-apps directory which is automatically added to the `PYTHONPATH` [in the Python container](https://github.com/cfpb/cfgov-refresh/blob/8c780180d5352fcd4d7f7ee63aee343fc8f91e9f/extend-environment.sh).

Adding satellite app clones to the Python path like this means that when cfgov-refresh code tries to import the [`paying_for_college`](https://github.com/cfpb/college-costs/tree/master/paying_for_college) app, say, it'll work. But if that app then tries to import `yaml`, it'll fail, because PyYAML was never installed in the Python environment.

A better way forward (that has been discussed a bit in other venues) might be to recommend that folks in Docker use `pip install -e` to install satellite apps ([as we do](https://cfpb.github.io/cfgov-refresh/development-tips/#tip-loading-satellite-apps) for the non-Docker setup). This would install the satellites' dependencies into the environment properly, and eliminate the requirement to duplicate them here. I would support that change but it's probably out of scope for this particular PR. For now perhaps we could just add a comment for dependencies like PyYAML (like we already have for django-mptt) indicating why it remains.

Reply:
That makes sense. Thanks for the explanation. I added PyYAML back in, with a comment, as you suggested.